prac_05/word_occurrences.py

```diff
@@ -0,0 +1,22 @@
+"""
+Word Occurrences
+Estimate: 20 minutes
+Actual:   28 minutes
+
+"""
+
+word_to_count = {}
+text = input("Text: ")
+words = text.split()
+for word in words:
+    word_frequency = word_to_count.get(word, 0) #.get returns keyname/value doesnt exist yet
+    word_to_count[word] = word_frequency + 1
+
+words = list(word_to_count.keys())
```

Comment:
Unsure why you remade the "words" list here from the dictionary... you already have a words list (line 10). Might be inefficient.